src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeCustomAttributeData.cs

```diff
@@ -1057,12 +1058,29 @@ internal sealed class CustomAttributeCtorParameter(CustomAttributeType type)
         public CustomAttributeEncodedArgument? EncodedArgument { get; set; }
     }
 
-    internal sealed class CustomAttributeNamedParameter(MemberInfo memberInfo, CustomAttributeEncoding fieldOrProperty, CustomAttributeType type)
+    internal sealed class CustomAttributeNamedParameter
     {
-        public MemberInfo MemberInfo => memberInfo;
-        public CustomAttributeType CustomAttributeType => type;
-        public CustomAttributeEncoding FieldOrProperty => fieldOrProperty;
+        public CustomAttributeNamedParameter(MemberInfo memberInfo, CustomAttributeEncoding fieldOrProperty, CustomAttributeType type)
+        {
+            MemberInfo = memberInfo;
+            Name = MemberInfo.Name;
```

Comment:
Is this change to cache the Name still needed?

Reply:
Nope, good point. Let me revert that.